stores/ini/store.go

```diff
@@ -24,7 +24,8 @@ func NewStore(c *config.INIStoreConfig) *Store {
 }
 
 func (store Store) encodeTree(branches sops.TreeBranches) ([]byte, error) {
-	iniFile := ini.Empty()
+	iniFile := ini.Empty(ini.LoadOptions{AllowNonUniqueSections: true})
+	iniFile.DeleteSection(ini.DefaultSection)
```

Comment:
What's the reason to delete the default section?

Reply:
`ini.Empty()` will already create a default section. Due to the `AllowNonUniqueSection` option set to true `iniFile.NewSection()` will create another one resulting in 2 empty default sections. Usually, an empty default section will be ignored by `iniFile.WriteTo()` but only when it is at the first position. Thus, the second empty default section will be included in the output although there is no default section specified in the input file.

As the `branches` parameter already includes a default section it will be recreated with `iniFile.NewSection()`.